packages/wrangler/src/init.ts

```diff
@@ -55,6 +56,69 @@ interface InitArgs {
 	site?: boolean;
 	yes?: boolean;
 }
+// TODO Move into types file?
```

Comment:
Which types file does this refer to?
Let's resolve this question in this PR and remove the comment.

Reply:
I'll just leave them in that file for now.